### PR TITLE
Handle fetch metadata server error

### PR DESCRIPTION
**Pull request by jfquevedol2198**
image

It redirects to scan page in 2 seconds

### CODE REVIEW

**Comment by meandrewandyou** [comment]
Looks like there's no designs for this, so maybe writing at the bottom could be changed to just `Redirecting...`?

File: scanner/src/routes/q/[iss]/[product]/[id]/+page.svelte
Comment (lietu) [inline scanner/src/routes/q/[iss]/[product]/[id]/+page.svelte:69]:
```suggestion
    <p>Returning to scanning mode...</p>
```
End-users don't care about "redirects" or "scan pages"

**Review by lietu — APPROVED** [review]
Since this is built on the other PRs, cannot merge this until those have been approved and merged - please remind me *on Slack* once this can be cleanly merged

**Pull request merged**
